authentik/lib/expression/evaluator.py

```diff
@@ -216,6 +220,74 @@ def expr_create_jwt(
         access_token.save()
         return access_token.token
 
+    def expr_send_email(
+        self,
+        address: str,
+        subject: str,
+        body: str | None = None,
+        stage: "EmailStage | None" = None,
+        template: str | None = None,
+        context: dict | None = None,
+    ) -> bool:
+        """Send an email using authentik's email system
+
+        Args:
+            address: Email address to send to
+            subject: Email subject
+            body: Email body (plain text/HTML). Mutually exclusive with template.
+            stage: EmailStage instance to use for settings. If None, uses global settings.
+            template: Template name to render. Mutually exclusive with body.
+            context: Additional context variables for template rendering.
+
+        Returns:
+            bool: True if email was queued successfully, False otherwise
+        """
+        # Deferred imports to avoid circular import issues
+        from authentik.stages.email.tasks import send_mail, send_mails
+        from authentik.stages.email.utils import TemplateEmailMessage
```

Comment:
This is a tiny bit annoying, since it might reduce the policy execution efficiency

Reply:
Agree, but I don't see other option. I updated the code and only import here `send_mail` and `send_mails`